addons/cds-connector-0.0.1/docs/overview.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Overview
+type: Overview
+---
+
+Use **CDS Connector** to connect your CDS backend with Kyma events to trigger serverless compute and make API calls from a lambda or a microservice.
+
+**Prerequisite**
+
+It uses the API token for makinng API calls to CDS. Please provide the clientId and clientSecret in the form parameters field in following format 'client_id=id,grant_type=client_credentials,client_secret=secret'
+
+After adding this addon in the namespace, copy the http url of event-gw lambda  and provide it in the lambda running on CDS.
```

Comment:
Be aware of that the addons repo here is actually used. So even if declared as Preview we should assure that the documentation is good and not misleading.

Can you please improve the section here to be more specific. There will be a lambda deployed as part of the addon called "event-gw"? and I should copy the URL of the lambda? Please see also docs from other connectors as a reference. Feel free to ask @bszwarc for help on that